Author: Rafael Diniz <rafael@rhizomatica.org>
Description: 
 remove ATTRIBUTE_UNUSED of the parameter uucp-org.patch uses
 add the possibility to send the system name to be called in pipe command
 (use \Z as place holder for the system name)
--- a/unix/pipe.c
+++ b/unix/pipe.c
@@ -226,8 +226,8 @@ static boolean
 fspipe_dial (qconn, puuconf, qsys, zphone, qdialer, ptdialer)
      struct sconnection *qconn;
      pointer puuconf;
-     const struct uuconf_system *qsys ATTRIBUTE_UNUSED;
-     const char *zphone ATTRIBUTE_UNUSED;
+     const struct uuconf_system *qsys;
+     const char *zphone;
      struct uuconf_dialer *qdialer;
      enum tdialerfound *ptdialer;
 {
@@ -253,6 +253,11 @@ fspipe_dial (qconn, puuconf, qsys, zphon
     if (!strcmp(*p, "\\H"))
       *p = zphone;
 
+  /* Look for a string \Z and replace by the system name to be called */
+  for (p=pzprog; *p; p++)
+    if (!strcmp(*p, "\\Z"))
+      *p = qsys->uuconf_zname;
+
   aidescs[0] = SPAWN_WRITE_PIPE;
   aidescs[1] = SPAWN_READ_PIPE;
   aidescs[2] = SPAWN_NULL;
